server/src/main/java/org/opensearch/search/internal/ContextIndexSearcher.java

```diff
@@ -283,6 +287,7 @@ public void search(
 
     @Override
     protected void search(List<LeafReaderContext> leaves, Weight weight, Collector collector) throws IOException {
+        TracerManager.getTracer().startSpan("IndexSearcher", Level.TERSE);
```

Comment:
Is this change accidental? I would expect this as a separate PR

Reply:
Agree. Will move it to separate PR as its logically different